fix gha ci

vicentebolea opened the pull request:
- **ci: correct conda settings**
- **MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.16.41.21**

vicentebolea committed:
ci: correct conda settings

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

vicentebolea committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.16.41.21

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.
